plugins/machina-tracker.js

```diff
@@ -30,13 +30,16 @@ machinaTracker.events = [];
 machinaTracker.setup = () => {
   $('<style>').prop('type', 'text/css').html('@include_string:machina-tracker.css@').appendTo('head');
 
-  var iconImage = '@include_img:images/marker-machina.png@';
-
-  machinaTracker.icon = L.icon({
-    iconUrl: iconImage,
-    iconSize: [26, 32],
-    iconAnchor: [12, 32],
-  });
+  var iconImage = '@include_img:images/marker-machina.svg@';
+
+  machinaTracker.icon = new (L.Icon.Default.extend({
+    options: {
+      iconUrl: iconImage,
+      iconSize: [26, 32],
+      iconAnchor: [12, 32],
+      shadowAnchor: [12, 41],
```

Comment:
all the defaults are already defined - these values make machina marker smaller than markers from classic player tracker